src/BizHawk.Client.Common/movie/MovieSession.cs

```diff
@@ -62,6 +62,17 @@ public IMovieController GenerateMovieController(ControllerDefinition definition
 			return new Bk2Controller("", definition ?? MovieController.Definition);
 		}
 
+		// Hack only used for TAStudio when starting a new movie
+		// This is due to needing to save a "dummy" default.tasproj
+		// This dummy file's initial save bypasses the normal queue/run
+		// new movie code (which normally sets the controller), although
+		// once it saves it goes through the normal queue/run code anyway.
+		// TODO: Stop relying on this dummy file so we do not need this ugly hack
+		public void SetMovieController(ControllerDefinition definition)
+		{
+			MovieController = new Bk2Controller(definition);
+		}
+

```

Comment:
Use the above helper? idk

Reply:
Given the todo ini that helper that might not be appropriate.

Reply:
Actually LogKey should just be null at this point, so the helper function should still operate fine regardless if that TODO is implemented.